src/main/java/com/booksaw/betterTeams/integrations/placeholder/TeamPlaceholderOptionsEnum.java

```diff
@@ -30,15 +30,15 @@ public enum TeamPlaceholderOptionsEnum {
 	/**
 	 * Constructor to take in an interface per enum value
 	 */
-	TeamPlaceholderOptionsEnum(IndividualTeamPlaceholderProvider teamProvider) {
+    TeamPlaceholderOptionsEnum(IndividualTeamPlaceholderProvider teamProvider) {
```

Comment:
these should be indented by a tab rather than 4 spaces to match the rest of the file